__setup__/enzyme.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies */
 
-import Enzyme from 'enzyme';
```

Comment:
So we don't use enzyme anymore? Do you plan to remove the file?

Reply:
Yeah, I'll remove it.